code/game/objects/items/weapons/melee/energy_melee_weapons.dm

```diff
@@ -27,7 +41,15 @@
 	force_off = initial(force) //We want to check this only when initializing, not when swapping, so sharpening works.
 	throwforce_off = initial(throwforce)
 
-/obj/item/melee/energy/attack(mob/living/target, mob/living/carbon/human/user)
+/obj/item/melee/energy/attack(mob/living/target, mob/living/user)
+	// Because for some reason the cleaving saw is an energy blade.
+	if(!istype(src, /obj/item/melee/energy/cleaving_saw))
```

Comment:
Could this just be shifted to an override on the cleaving saw, so we don't have  an `istype(src)` call

Reply:
There's now a check for it in cig code, turns out overriding attack didn't work.